storage/store.go

```diff
@@ -901,8 +932,8 @@ func (s *Store) processRaft(r raftInterface, closer chan struct{}) {
 			r, ok := s.ranges[raftCmd.cmd.RaftID]
 			s.mu.Unlock()
 			if !ok {
-				log.Errorf("got committed raft command for %d but have no range with that ID",
-					raftCmd.cmd.RaftID)
+				log.Errorf("got committed raft command for %d but have no range with that ID: %+v",
```

Comment:
I did indeed.  It's one of the reasons this took so long to get this in.  There was a bug in raft that would nondeterministically fail as it was trying to perform some lingering operation against the merged range.